src/magic_folder/web.py

```diff
@@ -357,6 +358,23 @@ def list_all_sanpshots(self, request):
         _application_json(request)
         return json.dumps(dict(_list_all_snapshots(self._global_config)))
 
+    @app.route("/magic-folder/<string:folder_name>/scan", methods=['PUT'])
+    @inline_callbacks
+    def scan_folder(self, request, folder_name):
+        try:
+            folder_service = self._global_service.get_folder_service(folder_name)
+        except KeyError:
+            returnValue(NoResource(b"{}"))
```

Comment:
this ends up doing the wrong thing (i believe you said earlier) (i.e. HTML instead of json)

Reply:
Yes, good catch. I guess I must have written this before I discovered that issue.